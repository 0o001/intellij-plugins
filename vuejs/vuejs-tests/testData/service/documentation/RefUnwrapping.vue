<script setup lang="ts">
import {ref} from "vue";

/**
 * docs have to be removed!
 */
let local = ref(323 as number | string);
</script>

<template>
  <div v-text="<caret>local"></div>
</template>