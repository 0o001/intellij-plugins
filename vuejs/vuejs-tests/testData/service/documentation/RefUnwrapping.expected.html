(property) local: string | number<br/>[ide]<br/>let local: number | string